themes/origins/assets/component-product-form.js

```diff
@@ -50,6 +48,12 @@ if (!customElements.get("yc-product-form")) {
       const attachedImage = this.attachedImage || null;
       const quantity = this.quantityValue || 1;
 
+      this.checkoutType === "express-checkout"
+        ? this.placeOrder(productVariantId, attachedImage, quantity)
```

Comment:
better if we explicitly pass what the method needs
```suggestion
        ? this.placeOrder(event.target, productVariantId, attachedImage, quantity)
```